acestep/api_server.py

```
@@ -2806,6 +3097,1607 @@ async def format_input_endpoint(request: Request, authorization: Optional[str] =
+            if adapter_name:
+                result = handler.add_lora(request.lora_path, adapter_name=adapter_name)
+            else:
+                result = handler.load_lora(request.lora_path)
+
+            if result.startswith("✅"):
+                response_data = {"message": result, "lora_path": request.lora_path}
+                if adapter_name:
+                    response_data["adapter_name"] = adapter_name
+                return _wrap_response(response_data)
+            else:
+                raise HTTPException(status_code=400, detail=result)
```

Ruff (0.15.0)

[warning] Abstract `raise` to an inner function

(TRY301)